packages/pangea-sdk/pangea/services/authn/models.py

```diff
@@ -0,0 +1,223 @@
+    email: str
+
+
+class UserDeleteResult(PangeaResponseResult):
+    # https://dev.pangea.cloud/docs/api/authn#delete-a-user
+    pass
+
+
+class UserInviteRequest(APIRequestModel):
+    inviter: str
+    email: str
+    callback: str
+    state: str
+    invite_ord: Optional[str] = None
+    require_mfa: Optional[bool] = None
+
+
+class UserInviteResult(PangeaResponseResult):
+    id: str
+    inviter: str
+    invite_org: str
+    email: str
+    callback: str
+    state: str
+    require_mfa: bool
+    created_at: str
+    expire: str
+
+
+class UserInvite(APIResponseModel):
+    id: str
+    inviter: str
+    invite_org: str
+    email: str
+    callback: str
+    state: str
+    require_mfa: bool
+    created_at: str
+    expire: str
+
+
+class UserInviteListResult(PangeaResponseResult):
+    invites: List[UserInvite]
+
+
+class UserInviteDeleteRequest(APIRequestModel):
+    id: str
+
+
+class UserInviteDeleteResult(PangeaResponseResult):
+    pass
+
+
+class UserListRequest(APIRequestModel):
+    scopes: Scopes
+    glob_scopes: Scopes
+
+
+class User(APIRequestModel):
+    profile: Profile
+    identity: str
+    email: str
+    scopes: Scopes
+
+
+class UserListResult(PangeaResponseResult):
+    users: List[User]
+
+
+class UserLoginRequest(APIRequestModel):
+    email: str
+    secret: str
+    scopes: Optional[Scopes] = None
+
+
+class UserLoginResult(PangeaResponseResult):
+    token: str
+    id: str
+    type: str
+    life: int
+    expire: str
+    identity: str
+    email: str
+    scopes: Optional[Scopes] = None
+    profile: Profile
+    created_at: str
+
+
+class UserProfileGetRequest(APIRequestModel):
+    identity: Optional[str] = None
+    email: Optional[str] = None
+
+
+class UserProfileGetResult(PangeaResponseResult):
+    identity: str
+    email: str
+    profile: Profile
+    id_provider: str
+    mfa_providers: List[str]
+    require_mfa: bool
+    verified: bool
+    last_login_at: str
+    disable: Optional[bool] = None
+
+
+class UserProfileUpdateRequest(APIRequestModel):
+    profile: Profile
+    identity: Optional[str] = None
+    email: Optional[str] = None
+    require_mfa: Optional[bool] = None
+    mfa_value: Optional[str] = None
+    mfa_provider: Optional[str] = None
+
+
+class UserProfileUpdateResult(PangeaResponseResult):
+    identity: str
+    email: str
+    profile: Profile
+    id_provider: str
+    mfa_providers: List[str]
+    require_mfa: bool
+    verified: bool
+    last_login_at: str
+    disable: Optional[bool] = None
+
+
+class UserUpdateRequest(APIRequestModel):
+    identity: Optional[str] = None
+    email: Optional[str] = None
+    authenticator: Optional[str] = None
+    disabled: Optional[bool] = None
+    require_mfa: Optional[bool] = None
+
+
+class UserUpdateResult(PangeaResponseResult):
+    identity: str
+    email: str
+    profile: Profile
+    scopes: Optional[Scopes] = None
+    id_provider: IDProvider
+    mfa_provider: Optional[List[str]] = None
```

Comment:
minor typo, I think this should be:
```suggestion
    mfa_providers: Optional[List[str]] = None
```